client/blocks/product-selector/index.jsx

```diff
@@ -177,6 +179,53 @@ export class ProductSelector extends Component {
 		);
 	}
 
+	getIntervalDiscount( selectedProductSlug ) {
+		const { basePlansPath, currencyCode, intervalType, siteSlug } = this.props;
+
+		if ( ! basePlansPath ) {
+			return null;
+		}
+
+		const isYearly = 'yearly' === intervalType;
```

Comment:
I'm a bit concerned that if so far this component has been supporting more than 2 billing intervals, with this logic it ties us with "monthly" and "yearly" only. I guess that makes sense for now - we can accommodate for a 2yearly if we ever end up adopting that in Jetpack.

Reply:
Yes, I noticed this as well.

I think that since we have no 2yearly option now, let's leave it as is. In case a 2yearly option would be considered to be added, it would probably be a separate project - there's many more issues to address in such case.